lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -128,10 +129,6 @@ def call(env)
           # unless it has been already set by the underlying framework
           if status.to_s.start_with?('5') && request_span.status.zero?
             request_span.status = 1
-            # in any case we don't touch the stacktrace if it has been set
```

Comment:
so this one sets wrong stack traces and removing it means writing this one for each framework built on top of Rack, right?